src/python/pants/backend/python/dependency_inference/rules.py

```diff
@@ -210,10 +210,14 @@ async def infer_python_dependencies_via_imports(
 
     unowned_dependency_behavior = python_infer_subsystem.unowned_dependency_behavior
     if unowned_imports and unowned_dependency_behavior is not UnownedDependencyUsage.DoNothing:
+        unowned_imports_with_lines = [
+            f"{modname} ({request.sources_field.address.filename}:{detected_imports[modname]})"
+            for modname in sorted(unowned_imports)
```

Comment:
Maybe `module_name`? `modname` confused me and made me think of video game mods lol